city_scrapers/spiders/chi_ssa_27.py

```diff
@@ -0,0 +1,134 @@
+from datetime import datetime
+
+from city_scrapers_core.constants import (
+    ADVISORY_COMMITTEE, BOARD, COMMISSION, COMMITTEE, NOT_CLASSIFIED
+)
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from scrapy import Field, Item
+
+
+class ChiSsa27Spider(CityScrapersSpider):
+    name = "chi_ssa_27"
+    agency = "Chicago Special Service Area #27 Lakeview West"
+    timezone = "America/Chicago"
+    allowed_domains = ["lakeviewchamber.com"]
+    start_urls = ["https://www.lakeviewchamber.com/ssa27"]
+    minutes_list = []
+    location = {
+        "name": "Sheil Park",
+        "address": "3505 N. Southport Ave., Chicago, IL 60657",
+    }
+
+    def parse(self, response):
+        """   `parse` should always `yield` Meeting items. """
+        self.minutes_list = self.get_minutes_panel_items(response)
+        self._validate_locations(response)
+        commission_path = "div #content-232764 div.panel-body p"
+
+        for item in response.css(commission_path)[1:]:  # main
+            title = self._parse_title(item)
+            start = self._parse_start(item)
+            link = self._parse_links(item),
```

Comment:
The comma at the end of this is causing `links` to be included as a tuple with a list inside. Taking it out should fix it

Reply:
Yes, I was wondering why the tuple had appeared.  Before I saw this comment, I rewrote it differently.  It works well for me now.